.docker/nginx-default.conf

```diff
@@ -17,6 +17,34 @@ server {
         fastcgi_param SCRIPT_FILENAME $request_filename;
         fastcgi_intercept_errors on;
         fastcgi_pass fpm:9000;
+        fastcgi_param SYMFONY__API_URL https://api.elifesciences.org;
```

Comment:
shouldn't it be something local too like `api_dummy` for the next server? Especially given the timeouts for calls in case it gets used (accidentally or on purpose)

Reply:
Could be, this was just replicating what we had before. Since we are now running the API dummy could be switched. Sometimes it's useful to point it at the real thing, but we could make it an environment variable that defaults to the local API dummy.